test/fixtures/output/schema-registry/exporter/describe.golden

```diff
@@ -4,6 +4,6 @@
 | Subject Format | my-${subject} |
 | Context Type   | CUSTOM        |
 | Context        | mycontext     |
-| Config         | key1="value1" |
-|                | key2="value2" |
+| Config         | key1=value1   |
```

Comment:
Let's `git blame` and ask the person who wrote this if this is ok?

Reply:
@Clarence97 I think you added the schema exporter commands initially. Is this change ok?

Reply:
Yes, this should be fine